bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/MenuItem.java

```diff
@@ -1151,17 +1176,12 @@ LRESULT wmMeasureChild (long wParam, long lParam) {
 
 	if ((parent.style & SWT.BAR) != 0) {
 		if (parent.needsMenuCallback()) {
-			/*
-			 * Weirdness in Windows. Setting `HBMMENU_CALLBACK` causes
-			 * item sizes to mean something else. It seems that it is
-			 * the size of left margin before the text. At the same time,
-			 * if menu item has a mnemonic, it's always drawn at a fixed
-			 * position. I have tested on Win7, Win8.1, Win10 and found
-			 * that value of 5 works well in matching text to mnemonic.
-			 * NOTE: autoScaleUpUsingNativeDPI() is used to avoid problems
-			 * with applications that disable automatic scaling.
-			 */
```

Comment:
Isn't this relevant anymore, so that the width can be set to a different value?

Reply:
This information still seems relevant on explaining the difference between `left margin` and `item size`. Also we are back to using value of 5 again to match the margins between items in menu bar. I would suggest to keep this comment as is.